AutowareAuto/src/perception/filters/point_cloud_fusion_nodes/launch/point_cloud_fusion.launch.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020-2021, The Autoware Foundation
```

Comment:
Since this file has solely been created for our team's usage, let's replace the whole Autoware Foundation copyright block (lines 1-15) with the following:

```
/*
 * Copyright (C) 2024 LEIDOS.
 *
 * Licensed under the Apache License, Version 2.0 (the "License"); you may not
 * use this file except in compliance with the License. You may obtain a copy of
 * the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
 * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 * License for the specific language governing permissions and limitations under
 * the License.
 */
```

Reply:
Done!